src/jetson/AES.py

```diff
@@ -8,23 +8,39 @@
 THRES = 400
 
 class encryption():
-    
+
     def __init__(self):
-        self.salt = os.urandom(16) #Salt variable
+        '''
+        This class handles encryption to prevent identifiable information (facial data)
+        from leaving the camera. It also generates a random key that will be used by
+        authorized personnel to acces the data.
+        '''
+        self.salt = os.urandom(16) #Salt variable (Generates a random byte string)
         self.key = PBKDF2("passphrase", self.salt).read(16) #Creates key using KDF scheme
-        
+
     def encrypt(self, coordinates, image):
+        '''
+        This method encrypts the facial regions.
+
+        Params-
+        coordinates: bounding box coordinates
+        image: Image to be encrypted
+
+        Returns-
+        image: Encrypted image
+        IV: Initialization vector for decrypting image
+        '''
```

Comment:
Please provide types for these parameters in both the function definition as [type hinting](https://www.python.org/dev/peps/pep-0484/#type-definition-syntax) and in the function docstring.